### PR TITLE
fix: edit button does not flash when adding new column

### PR DESCRIPTION
**Pull request by jarekdanielak**
### Proposed Changes

Fix for flashing edit button when adding a new column.

Related https://github.com/camunda/camunda-modeler/issues/4388

https://github.com/user-attachments/assets/259bc300-69e2-49cc-841c-0861e4a37a6d

 

### Checklist

To ensure you provided everything we need to look at your PR:

* [X] **Brief textual description** of the changes present
* [X] **Visual demo** attached
* [ ] **Steps to try out** present, i.e. [using the `@bpmn-io/sr` tool](https://github.com/bpmn-io/sr)
* [X] Related issue linked via `Closes {LINK_TO_ISSUE}` or `Related to {LINK_TO_ISSUE}`

### CODE REVIEW

**Comment by nikku** [comment]
Tests are failing, likely due to your changes. Could you have a look?

**Comment by nikku** [comment]
Also, as a good practice, could you add a screen capture to show the improved interaction?

**Comment by barmac** [comment]
As a `fix`, this could be targetted at `main` instead of `develop`.

**Comment by barmac** [comment]
The edit button is not displayed anymore:

https://github.com/user-attachments/assets/f2787abc-b85a-44d9-b44e-3c7e1776a1cb

**Comment by barmac** [comment]
Compare:

https://github.com/user-attachments/assets/5fed2f7f-bdfd-4b03-ac26-6fde8bb4a5b4

**Review by barmac — CHANGES_REQUESTED** [review]
Let's fix the CSS before merging this.

File: packages/dmn-js-decision-table/assets/css/dmn-js-decision-table-controls.css
Comment (barmac) [inline packages/dmn-js-decision-table/assets/css/dmn-js-decision-table-controls.css:289]: Can this solution be [signed-off from the maintainer perspective](https://github.com/bpmn-io/internal-docs/blob/main/development/DEFINITION_OF_DONE.md)? What if we replace the plus icon with something else?

Comment (barmac) [reply packages/dmn-js-decision-table/assets/css/dmn-js-decision-table-controls.css:289]: Should the edit button visibility be driven by another component?

Comment (jarekdanielak) [reply packages/dmn-js-decision-table/assets/css/dmn-js-decision-table-controls.css]: Thanks, refactored.

**Review by barmac — APPROVED** [review]

**Comment by barmac** [comment]
When merging this, please squash the changes.

**Pull request merged**
